classes/03class/exercises/c03-docker01/henriqueediniz/c03-docker01.txt

```diff
@@ -0,0 +1,39 @@
+👽 henriquediniz docker build -t da-terraform:c03-docker-e01 -f Dockerfile .
+Sending build context to Docker daemon   2.56kB
+Step 1/4 : FROM alpine:3.9.5
+3.9.5: Pulling from library/alpine
+9123ac7c32f7: Pull complete 
+Digest: sha256:115731bab0862031b44766733890091c17924f9b7781b79997f5f163be262178
+Status: Downloaded newer image for alpine:3.9.5
+ ---> 82f67be598eb
+Step 2/4 : WORKDIR /Users/henriquediniz/downloads
+ ---> Running in 08af3ea8563c
+Removing intermediate container 08af3ea8563c
+ ---> ed83a78a5bd4
+Step 3/4 : RUN apk update   && apk add --no-cache wget unzip   && wget https://releases.hashicorp.com/terraform/0.12.20/terraform_0.12.20_linux_amd64.zip --output-file terraform.zip   && unzip ./terraform_0.12.20_linux_amd64.zip -d /usr/local/bin/   && rm -f terraform.zip   && apk del wget unzip
+ ---> Running in c8ba732f2076
+fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
+fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
+v3.9.5-18-g276b549fe5 [http://dl-cdn.alpinelinux.org/alpine/v3.9/main]
+v3.9.5-14-g437bc75312 [http://dl-cdn.alpinelinux.org/alpine/v3.9/community]
+OK: 9774 distinct packages available
+fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/main/x86_64/APKINDEX.tar.gz
+fetch http://dl-cdn.alpinelinux.org/alpine/v3.9/community/x86_64/APKINDEX.tar.gz
+(1/2) Installing unzip (6.0-r4)
+(2/2) Installing wget (1.20.3-r0)
+Executing busybox-1.29.3-r10.trigger
+OK: 6 MiB in 16 packages
+Archive:  ./terraform_0.12.20_linux_amd64.zip
+  inflating: /usr/local/bin/terraform  
+(1/2) Purging unzip (6.0-r4)
+(2/2) Purging wget (1.20.3-r0)
+Executing busybox-1.29.3-r10.trigger
+OK: 6 MiB in 14 packages
+Removing intermediate container c8ba732f2076
+ ---> 9964ad2dd94d
+Step 4/4 : CMD ["terraform", "-v"]
+ ---> Running in 464cd167a919
+Removing intermediate container 464cd167a919
+ ---> 3012a472bd0b
+Successfully built 3012a472bd0b
+Successfully tagged da-terraform:c03-docker-e01
```

Comment:
can you also execute 
`docker images` and provide its output so we can confirm that the image has the required size?

Reply:
output result in this PR https://github.com/devopsacademyau/academy/pull/380